Projects.toml

```diff
@@ -0,0 +1,6 @@
+[parity-processbot]
```

Comment:
How will the bot get a list of projects that have this file? Should we also include a kind of repository manifest in `processbot` that lists the repos it's enabled in? Right now we go through every directory available, but that might be how we want to roll this out. We might want to start with one or two repositories get feedback and enabled it more over time.

Reply:
We could either add a list of enabled repos or poll them all and only act on those with the file.  I lean towards the later so that adding/removing a repo doesn't require a PR on processbot.